includes/helpers.php

```diff
@@ -5,3 +5,20 @@ function ctype_alnum( $text ){
 		return ( preg_match('~^[0-9a-z]*$~iu', $text ) > 0 );
 	}
 }
+
+if ( ! function_exists( 'update_entry_meta' ) ) {
+
+	/**
+	 * Update Entry meta
+	 *
+	 * Allows a meta to be altered before posting to Eloqua.
+	 *
+	 * @param  string $key  The key of the metadata.
+	 * @param  string $lead The value of the metadata.
+	 * @param  object $form The form that the metadata belongs to.
+	 * @return string       Returns the value of the entry meta.
+	 */
+	function update_entry_meta( $key, $lead, $form ) {
```

Comment:
@solepixel the function that calls that logic `add_success_meta` is hooked into the `form_update_meta` action:

`add_action( 'gform_entry_meta', array( $this, 'add_success_meta' ), 10, 2 );`